bin/batch-bag

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [[ -z "$2" ]]; then
+  echo "Usage: batch-bag <audio|video> <directory> [additional_directory...]"
+  exit 1
+fi
+
+if [[ -n "$CHIPMUNK_HOME" ]]; then
+  if [[ -x "$CHIPMUNK_HOME/bin/makebag" ]]; then
+    MAKEBAG="$CHIPMUNK_HOME/bin/makebag"
+  else
+    echo "If CHIPMUNK_HOME is set, it must point to a chipmunk-client directory with bin/makebag in it."
+    exit 1
+  fi
+elif [[ ! $(which makebag) ]]; then
```

Comment:
Is the advantage of `[[ ! $(which makebag) ]]` over `! which makebag > /dev/null` just that you don't have to do anything with stdout?

Reply:
I almost always use subshell for checking exit codes... Maybe the inlined / redirected pattern is better? Wouldn't you have to check `$?` explicitly? Anyway, it works as-is.